extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -160,60 +180,76 @@ public function getSimpleLanguageCode() {
 
 	/**
 	 * Allows us to abstract out the MW Api for parse responses
+	 *
 	 * @param int $pageId
+	 *
 	 * @return array
 	 */
 	public function getParseResponseFromPageId( $pageId ) {
-		return \ApiService::call( array(
-					'pageid'	=> $pageId,
-					'action'	=> 'parse',
-				));
+		return \ApiService::call(
+			[
+				'pageid' => $pageId,
+				'action' => 'parse',
+			]
+		);
 	}
 
 	/**
 	 * Performs an API request, but still returns the data
+	 *
 	 * @param int $pageId
+	 *
 	 * @return Mixed
 	 **/
 	public function getBacklinksCountFromPageId( $pageId ) {
 		$title = $this->getTitleStringFromPageId( $pageId );
-		$data = \ApiService::call( array(
-				'titles'	=> $title,
-				'bltitle'	=> $title,
-				'action'	=> 'query',
-				'list'		=> 'backlinks',
-				'blcount'	=> 1
-				));
-		return isset($data['query']['backlinks_count'] ) ? $data['query']['backlinks_count'] : 0;
+		$data = \ApiService::call(
+			[
+				'titles' => $title,
+				'bltitle' => $title,
+				'action' => 'query',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "query" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)